types/npm-package-json-lint/index.d.ts

```diff
@@ -0,0 +1,53 @@
+// Type definitions for npm-package-json-lint 5.1
+// Project: https://github.com/tclindner/npm-package-json-lint
+// Definitions by: Hannes Leutloff <https://github.com/yeldiRium>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.3
```

Comment:
```suggestion
```
Do you really want to restrict this type to newest TS version only? (there seems to be no 4.3 only features used in the code)

Reply:
Oh i just used the most recent because I use it for everything. I didn't realize this limits the usage of the package. Can you recommend a more fitting version? I'm not sure what all the breaking changes between the major versions were.

Reply:
just skip versioning, unless there is something in package definition that relies on specific version (e..g like `export type { .... } ...` -> 3.8, etc)